flow_test/manual_dask_flow.py

```diff
@@ -0,0 +1,72 @@
+import json
+import os
+
+import prefect
+import yaml
+from prefect import Flow, storage, task
+from prefect.engine.executors import DaskExecutor
+from prefect.run_configs import ECSRun
+
+from flow_test.utils import generate_tags, retrieve_stack_outputs
+
+project = os.environ["PREFECT_PROJECT"]
+worker_image = os.environ["PREFECT_DASK_WORKER_IMAGE"]
+flow_name = "dask-test-flow"
+
+definition = yaml.safe_load(
+    """
+    networkMode: awsvpc
+    cpu: 1024
+    memory: 2048
+    containerDefinitions:
+        - name: flow
```

Comment:
Is there a risk of this being too generic if we run multiple flows? I'm not sure what the behaviour of this property is.